GameServerLib/Packets/PacketHandlers/HandleAttentionPing.cs

```diff
@@ -1,11 +1,11 @@
 ﻿using GameServerCore;
 using GameServerCore.Packets.Enums;
```

Comment:
Does ``GameServerLib`` still need to know about ``GameServerCore.Packets.Enums`` at this point?
I don't think this should be the case as those should be in ``PacketsDefinition420`` ?

Reply:
4.2 packet uses only Alpha as float `public SetScreenTint(TeamId team, bool enable, float transitionTime, byte red, byte green, byte blue, float alpha)`, so Im not sure how to convert byte <-> float.

Reply:
No it doesn't need to, but because of unused `PacketCmd PacketType` that it sets we have this reference, Ill remove all such references and from the handler base in Core.

Reply:
Fixed the unused variable.